### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [HK-SHAO/HK-SHAO.github.io#16] [General]
# about/

烧风的个人网站

https://shao.fun/about/

**Comment by HK-SHAO** [comment]
在瞅啥，不如给我发条消息试试！😘

**Reply by duskdusts** [reply]
发条消息试试

**Reply by HK-SHAO** [reply]
好椰

**Comment by aahowe** [comment]
hi😃

**Reply by HK-SHAO** [reply]
halo

**Comment by rantrism** [comment]
您好～我是腾讯云开发者社区运营，关注了您分享的技术文章，觉得内容很棒，我们诚挚邀请您加入腾讯云自媒体分享计划。完整福利和申请地址请见：https://cloud.tencent.com/developer/support-plan
作者申请此计划后将作者的文章进行搬迁同步到社区的专栏下，你只需要简单填写一下表单申请即可，我们会给作者提供包括流量、云服务器等，另外还有些周边礼物。

**Comment by xiaoyxue** [comment]
昨天微信加你了

**Comment by w1ndys** [comment]
萌备异次元来的，膜拜大佬，太强了大佬

**Reply by HK-SHAO** [reply]
还有这操作（惊讶
